public/app/core/utils/navBarItem-translations.ts

```diff
@@ -89,6 +89,8 @@ export function getNavTitle(navId: string | undefined) {
       return t('nav.alerting-groups.title', 'Active notifications');
     case 'alerting-admin':
       return t('nav.alerting-admin.title', 'Settings');
+    case 'alerts-trash':
+      return t('nav.alerts-trash.title', 'Trash');
```

Comment:
TODO: remove

Reply:
I have to update this , as now we use alerts/recently-deleted as the id in the navtree